wiki/Alphabet.md

```diff
@@ -0,0 +1,183 @@
+---
+title: History and replacement of Bio.Alphabet
+permalink: wiki/Alphabet
+layout: wiki
+tags:
+ - Wiki Documentation
+---
+
+# Introduction
+
+This page is intended to help people updating existing code using Biopython to
+cope with the removal of the ``Bio.Alphabet`` module in Biopython 1.78
+(September 2020).
+
+The objects from `Bio.Alphabet` had two main uses:
+* Record the molecule type (DNA, RNA or protein) of a sequence,
+* Declare the expected characters in a sequence, alignment, motif, etc.
+
+# Motivation
+
+There were drawbacks to the twenty year old design. In particular how the
```

Comment:
I think the bigger issue is that the purpose of alphabets was not defined. Sure, the implementation was also problematic, but the implementation could have been fixed if we had been able to figure out what alphabets were supposed to do in the first place.

Reply:
Precede this with "The intended purpose of the alphabet objects was never clearly defined, and ..."?

Reply:
ok